lib/experimental/Widgets/Widget/index.tsx

```diff
@@ -46,6 +46,7 @@ export interface WidgetProps {
     text: string
     variant: StatusVariant
   }
+  fullHeight?: boolean
```

Comment:
I think the right way to make things full height is for the container to actually dictate that instead of the children. We can easily do:

```
<div class="flex flex-row items-stretch">
  <Widget ... />
</div>
```

What do you think?